wolox-training/ap-swift
# Library data

## Summary ##
**Trello card**: https://trello.com/c/oBIzM5n0/13-vista-de-biblioteca
**Trello card**: https://trello.com/c/SvEAkxWU/14-tabla-paginada-de-libros

Request book list from API and used data to show booktableview with name, author and image for each book cell.

## Test Plan ##

  * Open the app
  * Login with google
  * The main tab bar view should appear after a successfull login.
  * Book table should be displayed after a while

## Screenshots ##

### iPhone 7s
![login-view](https://cloud.githubusercontent.com/assets/25587882/23507935/1d3ba7d0-ff2e-11e6-9b3e-90f853d8a9d4.png)
![libraryview](https://cloud.githubusercontent.com/assets/25587882/23507938/203897e0-ff2e-11e6-8f0e-e9636bf2c9fb.png)

## Screencast ##

![libraryview](https://cloud.githubusercontent.com/assets/25587882/23507892/e09ffae2-ff2d-11e6-9e59-4a60ea7b0964.gif)
